libraries/ESP8266WebServer/src/ESP8266WebServer.cpp

```diff
@@ -440,7 +440,7 @@ void ESP8266WebServer::sendContent(const String& content) {
   if(_chunked) {
     char * chunkSize = (char *)malloc(11);
```

Comment:
Would this be an appropriate spot to reduce some technical debt?  This is a silly `malloc` whose pointer takes an appreciable amount of memory as the actual data itself?

Changing to `char chunkSize[11];` and dropping the check for `chunkSize!=NULL`and the `free(chunkSize);` would seem to me to be trivial enough.